### DISCUSSION TITLE
browse.GetRecommendations

### DISCUSSION THREAD

**Discussion by FreddyE1982** [JohnnyCrazy/SpotifyAPI-NET#602] [Q&A]
Could you post an example on how to use "browse" (for example browse.GetRecommendations)", please? I just can´t seem to figure it out ;-)

**Comment by bnoffer** [comment]
Same here, I am unable to figure out how to populate the RecommendationsRequest since everything is read only. Found a initialization in a older issue ( #311 ) ticket here on Github, but that initialization does not work (anymore?).

**Reply by FreddyE1982** [reply]
I found out how to do it. I didn't know that readonly properties can have methods. 😅

Create a new instance of RecommendationsRequest. It has the property "SeedTracks" containing the method "add".

For VB.net this would be:

Dim myRequest as New RecommendationsRequest

myRequest.SeedTracks.add(spotifyID1)
myRequest.SeedTracks.add(spotifyID2)

etc

After that you can run the request like this:

Dim similar = await yourspotifyclient.Browse.GetRecommendations(myRequest)

**Reply by bnoffer** [reply]
ah, thank you very much, I could have thought about that myself. As specially that now when I see it I remember using the same concept a few days back.
